src/odemis/driver/smarpod.py

```diff
@@ -350,29 +349,34 @@ def __init__(self, name, role, locator, axes=None, **kwargs):
         # will take care of executing axis move asynchronously
         self._executor = CancellableThreadPoolExecutor(1)  # one task at a time
 
-        # Check referencing, and reference if not referenced
+        axes_ref = {a: False for a, i in self.axes.items()}
```

Comment:
Set it to `referenced.value` at init.

Reply:
Although referenced.value is defined from this...?

Reply:
It's confusing... I don't mean `self.referenced.value`, I mean the output argument  to `SmarPod_IsReferenced()`, which is also called `referenced`.